File: Python/calculator.py
# Simple CLI Calculator

operations = "1. Addition 2. Subtraction 3. Division 4. Multiplication 5. Modulus"
user_choice = input(f"Enter a Choice:\n {operations}\n")

number_1 = eval(input("Enter Number 1:"))
number_2 = eval(input("Enter Number 2:"))

if user_choice == '1':
    addition = number_1 + number_2
    print(f"The Result of {number_1} + {number_2} is:{addition}")
elif user_choice == '2':
    subtraction = number_1 - number_2
    print(f"The Result of {number_1} - {number_2} is:{subtraction}")
elif user_choice == '3':
    division = number_1 / number_2
    print(f"The Result of {number_1} / {number_2} is:{division}")
elif user_choice == '4':
    multiplication = number_1 * number_2
    print(f"The Result of {number_1} * {number_2} is:{multiplication}")
elif user_choice == '5':
    modulus = number_1 % number_2
    print(f"The Result of {number_1} % {number_2} is:{modulus}")
else:
    print("Invalid Choice. Try Again!")

File: Fullstack Web/Javascript/Calculator/calculator.js
/*
Build a basic arithmetic calculator without a frontend
The calculator should be able to perform basic operations like Addition, Subtraction, Multiplication, & Division
*/

// user input, two number and operation

let number = Number(prompt("Enter First Number:"));

var operations = prompt("Choose Operation:(+, -, *, /)");

let number_2 = Number(prompt("Enter Second Number:"));

// program logic and expression

if (operations == "+"){
    var result = number + number_2
    window.alert("Answer: " + result)
}
else if(operations == "-"){
    var result = number - number_2
    window.alert("Answer: " + result)
}
else if(operations == "*"){
    var result = number * number_2
    window.alert("Answer: " + result)
}
else if(operations == "/"){
    var result = number - number_2
    window.alert("Answer: " + result)
}